drivers/gles3/shaders/scene.glsl

```diff
@@ -565,6 +565,8 @@ in vec3 normal_interp;
 //used on forward mainly
 uniform bool no_ambient_light;
 
+uniform vec2 viewport_size;
```

Comment:
The best is to put this in the UBO above with all the global variables (including transforms, time and the other stuff). It may actually even be there already (i dont have access right now to my computer).